src/blocks/content-helper/content-helper-provider.ts

```diff
@@ -41,6 +41,9 @@ interface GetRelatedTopPostsResult {
 	posts: RelatedTopPostData[];
 }
 
+export const RELATED_POSTS_DEFAULT_LIMIT = 5;
+export const RELATED_POSTS_DEFAULT_TIME_RANGE = 3; // in days
```

Comment:
```suggestion
export const RELATED_POSTS_DEFAULT_TIME_RANGE = 3; // In days.
```

Let's attempt to respect WP commenting guidelines wherever we can. This gives consistency across all of our comments.